scripts/requirements-doc.txt

```diff
@@ -0,0 +1,5 @@
+sphinx==3.2.1
```

Comment:
I had to temporarily set sphinx here, because otherwise we use Zephyr's one which now is 3.3.0, but it is raising an exception building nrfxlib.